javaharness/README.md

```diff
@@ -60,9 +60,9 @@ Follow the steps to inspect and debug Arcs:
   adb wait-for-device root
   adb reverse tcp:8786 tcp:8786
   ```
-4. Instructing the on-device Arcs runtime to use ALDS proxy before starting it:
+4. Instructing the on-device Arcs runtime to connect to the Arcs Explorer tool:
 * ```bash
-  adb shell setprop debug.arcs.runtime.use_alds true
+  adb shell setprop debug.arcs.runtime.enable_arcs_explorer true
```

Comment:
Maybe not important for right now, but is there a more user-friendly way of changing these properties rather than using adb shell commands? They're not very intuitive..

Reply:
With @yuangu-google we're planning to have a user friendly script that sets the device the way you want. Maybe:

```bash
$ arcs-android-setup development
> Enabled port forwarding on 8786
> Enabled Developer Tools
> Enabled loading assets from the workstation
> Starting Arcs Development Server...
```

To be decided what exactly goes in there...